source/services/linkedin.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * LinkedIn service provider
+ */
+
+module.exports = {
+    counterUrl: 'http://www.linkedin.com/countserv/count/share?url={url}&format=jsonp&callback=?',
+    convertNumber: function (counter) {
+        return counter.count;
+    },
+    popupUrl: 'https://www.linkedin.com/shareArticle?url={url}&mini=true&title={title}',
+    popupWidth: 600,
+    popupHeight: 500
+};
```

Comment:
Thank you :). Right. I've created issue to discuss it with maintainers: https://github.com/ilyabirman/Likely/issues/17.